recipes/poppler/all/conanfile.py

```diff
@@ -1,15 +1,19 @@
-from conans import CMake, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd, cross_building
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rmdir
+from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.52.0"
 
 
 class PopplerConan(ConanFile):
     name = "poppler"
     description = "Poppler is a PDF rendering library based on the xpdf-3.0 code base"
     homepage = "https://poppler.freedesktop.org/"
-    topics = ("conan", "poppler", "pdf", "rendering")
+    topics = "poppler", "pdf", "rendering"
```

Comment:
The name does not need to be in the topic as it does not help with searching (which is the goal of topics)

```suggestion
    topics = ("pdf", "rendering")
```